app/_locales/pt_BR/messages.json

```
@@ -3,105 +3,180 @@
     "message": "Sobre"
   },
   "aboutSettingsDescription": {
-    "message": "Versão, centro de apoio e informações de contato"
+    "message": "Versão, central de suporte e informações de contato"
   },
   "acceleratingATransaction": {
-    "message": "* Acelerar uma transação usando um preço de gás mais alto aumenta suas chances de a rede processá-la de forma mais rápida, mas isso nem sempre é garantido."
+    "message": "* Acelerar uma transação usando um Gas Price mais alto aumenta as chances de agilizar o processamento pela rede, mas nem sempre é garantido."
+  },
+  "acceptTermsOfUse": {
+    "message": "Eu li e concordo com $1",
+    "description": "$1 is the `terms` message"
+  },
+  "accessAndSpendNotice": {
+    "message": "$1 pode acessar e gastar até esse valor máximo",
+    "description": "$1 is the url of the site requesting ability to spend"
   },
   "accessingYourCamera": {
-    "message": "Acessando a sua câmera..."
+    "message": "Acessando sua câmera..."
   },
   "account": {
     "message": "Conta"
   },
   "accountDetails": {
-    "message": "Detalhes da Conta"
+    "message": "Detalhes da conta"
   },
   "accountName": {
-    "message": "Nome da Conta"
+    "message": "Nome da conta"
   },
   "accountOptions": {
-    "message": "Opções da Conta"
+    "message": "Opções da conta"
   },
   "accountSelectionRequired": {
     "message": "Você precisa selecionar uma conta!"
   },
+  "active": {
+    "message": "Ativo"
+  },
+  "activity": {
+    "message": "Atividade"
+  },
   "activityLog": {
-    "message": "registro de atividade"
+    "message": "log de atividades"
+  },
+  "addAccount": {
+    "message": "Adicionar uma conta"
   },
   "addAcquiredTokens": {
-    "message": "Adicione os tokens que você adquiriu usando o MetaMask"
+    "message": "Adicionar os tokens que você adquiriu usando o MetaMask"
   },
   "addAlias": {
-    "message": "Adicionar pseudônimo"
+    "message": "Adicionar alias"
   },
   "addNetwork": {
-    "message": "Adicionar Rede"
+    "message": "Adicionar rede"
   },
   "addRecipient": {
-    "message": "Adicionar Destinatário"
+    "message": "Adicionar destinatário"
   },
   "addSuggestedTokens": {
-    "message": "Adicionar Tokens Sugeridos"
+    "message": "Adicionar tokens sugeridos"
   },
   "addToAddressBook": {
-    "message": "Adicionar à lista de endereços"
+    "message": "Adicionar ao catálogo de endereço"
   },
   "addToAddressBookModalPlaceholder": {
-    "message": "Exemplo: John D."
+    "message": "por ex., John D."
```

The correct abbreviation of `por exemplo` would be `p ex` (corresponding to e.g.). But it is common to see different usages (`Exemplo:`, `Ex.,`, etc.).

Source (pt-br dictionary): https://michaelis.uol.com.br/moderno-portugues/como-consultar/abreviaturas/

---

Thanks for the suggestion! I'll update this to `p ex` then.

I'm a little unclear on the punctuation :thinking: Does this look correct? Or should I remove the comma, or the periods?
```suggestion
    "message": "p. ex., John D."
```

---

I did a quick search (https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/ponto-no-final-da-frase-apos-ponto-da-abreviatura/31884) and it looks like the correct usage would be `p.ex.: John D.`.

I can't say if this is really the right way of using it, but I have definitely seen `p. ex.:` being commonly used.

---

Ok, well in that case I'll go with what you found:

```suggestion
    "message": "p.ex.: John D."
```

We can always change it later!